addon/components/au-alert.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+
+export default class AuAlertComponent extends Component {
+  get alertSkin() {
+    if (this.args.alertskin == "info")
+      return "--info";
+    if (this.args.alertskin == "success")
+      return "--success";
+    if (this.args.alertskin == "warning")
+      return "--warning";
+    if (this.args.alertskin == "error")
+      return "--error";
+    return "";
```

Comment:
Can a switch be a better candidate for this control structure?

Reply:
No clue honestly. We still need to have a decent conversation on what the best approach is here. This pattern will be reused in many places.